docs/pipelines/tinycap.md

```diff
@@ -72,6 +72,32 @@ One can specify other options such as: `bowtie` (alignment) options, clipping an
 }
 ```
 
+## Taxonomy extraction 
```

Comment:
I understand your point. However, there are some reasons why I think that may not be the best approach:

1. To understand all of the options of a pipeline, the user already has to click through several pages. Adding yet another would place yet more cognitive burden. Having all options pertaining to one pipeline in one document may look a bit cleaner.
2. Where to put such a stand-alone taxonomy extraction document is also not entirely clear. It's not a pipeline, so it wouldn't belong under `pipelines`. It's also not generalizable to _all_ pipelines, so placing it under `general` also feels wrong. 

That said, I do certainly understand the duplication isn't the best from a maintenance POV. Preferably, I'd like to import the section in all relevant pipelines. I'll look if such a thing is possible.